src/content/docs/mobile-monitoring/new-relic-mobile-android/install-configure/configure-proguard-or-dexguard-android-apps.mdx

```diff
@@ -25,21 +25,21 @@ To add support for [ProGuard](http://proguard.sourceforge.net/manual/usage.html)
    * To preserve stack information for handled exceptions, add `SourceFile` and `EnclosingMethod.`
 2. Add the following to your [ProGuard config file](#proguard-library) (`proguard.cfg`, `proguard-android.txt`, `proguard-rules.pro`, etc.):
 
-   ```
-   -keep class com.newrelic.** { *; }
-   -dontwarn com.newrelic.**
-   -keepattributes Exceptions, Signature, InnerClasses, LineNumberTable, SourceFile, EnclosingMethod
-
-  ##
-  ## NewRelic Gradle plugin 7.x may require the following additions:
-  ##
-  # Retain generic signatures of TypeToken and its subclasses if [R8 version 3.0 full-mode](https://r8.googlesource.com/r8/+/refs/heads/master/compatibility-faq.md#r8-full-mode) is enabled.
-  -keepattributes Signature
-  -keep class com.newrelic.com.google.gson.reflect.TypeToken { *; }
-  -keep class * extends com.newrelic.com.google.gson.reflect.TypeToken
-  # For using GSON @Expose annotation
-  -keepattributes *Annotation*
-   ```
+    ```
+    -keep class com.newrelic.** { *; }
+    -dontwarn com.newrelic.**
+    -keepattributes Exceptions, Signature, InnerClasses, LineNumberTable, SourceFile, EnclosingMethod
+
+    ##
+    ## NewRelic Gradle plugin 7.x may require the following additions:
+    ##
+    # Retain generic signatures of TypeToken and its subclasses if 8 version 3.0 full-mode (https://r8.googlesource.com/r8/+/refs/heads/master/compatibility-faq.md#r8-full-mode) is enabled.
```

Comment:
@brnhensley should we keep the "R" in "R8"? This seems less clear without that.

Reply:
I don't recall removing that intentionally, I think that's a typo. I just pushed a commit to correct this and make it a little nicer to look at too without losing the helpful link